doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -256,6 +256,9 @@ Wi-Fi
 -----
 
 * The WPA supplicant is now switched to Zephyr upstream's fork instead of |NCS|.
+* Added the :ref:`ug_nrf70_developing_offloaded_raw_tx` page.
+* Added support for EAP-TLS authentication in the Wi-Fi stack.
+* Added :ref:`Platform Security Architecture (PSA) APIs support <ug_nrf70_developing_wifi_psa_support>`  for WPA2 security profiles.
```

Comment:
```suggestion
* Added:

   * The :ref:`ug_nrf70_developing_offloaded_raw_tx` page.
   * Support for EAP-TLS authentication in the Wi-Fi stack.
   * Support for :ref:`Platform Security Architecture (PSA) APIs <ug_nrf70_developing_wifi_psa_support>`  for WPA2™ security profiles.

* Updated:

  * The WPA supplicant is now switched to Zephyr upstream's fork instead of |NCS|.
```